qiskit/test/base.py

```diff
@@ -197,9 +197,6 @@ def setUpClass(cls):
         for mod in allow_DeprecationWarning_modules:
             warnings.filterwarnings("default", category=DeprecationWarning, module=mod)
         allow_DeprecationWarning_message = [
-            r".*LogNormalDistribution.*",
-            r".*NormalDistribution.*",
-            r".*UniformDistribution.*",
```

Comment:
Thanks for removing these - it's good to get rid of stuff that's not used any more.